internal/characters/rosaria/skill.go

```diff
@@ -40,6 +37,12 @@ func (c *char) Skill(p map[string]int) action.ActionInfo {
 		HitlagFactor:       0.01,
 		CanBeDefenseHalted: false,
 	}
+	// We always assume that A1 procs on hit 1 to simplify
+	//TODO: does this need to change if we add player position?
```

Comment:
NIT: Move todo to the `a1()` function definition?.  Change todo to be about needing to add enemy direction in order to determine in front vs behind (or just remove TODO completely if we will never implement)